pkg/operator/ceph/controller/controller_utils.go

```diff
@@ -89,8 +88,8 @@ var (
 	OperatorCephBaseImageVersion string
 
 	// loopDevicesAllowed indicates whether loop devices are allowed to be used
-	loopDevicesAllowed          = false
-	revisionHistoryLimit *int32 = nil
+	loopDevicesAllowed   = false
+	revisionHistoryLimit *int32
```

Comment:
I don't care about this either way, but I do want to note that this change results in no functional difference at runtime. The zero value for all pointer types is nil. In the previous code, it was explicit, and in the new code, it's implicit.